lib/read_aligner.hh

```diff
@@ -204,8 +204,18 @@ private:
         return ret;
     }
 
+    Alignment * _empty_alignment() {
```

Comment:
EIther make this a subclass of Alignment and use a member initialization list or move to `read_aligner.cc`